### DISCUSSION TITLE
Assign role without team id

### DISCUSSION THREAD

**Discussion by webmastervinay** [spatie/laravel-permission#1853] [Q&A]
I am using spatie/laravel-permission v5 and enabled teams feature.

If it is enabled, does that mean that all the roles should have a team id? I checked the documentation. It says if team id is null then it will be a global role.

![image](https://user-images.githubusercontent.com/5024049/132939132-eab33dd4-7b0e-4b52-8b99-8c68a9bdc76f.png)

But in a fresh laravel application, assigning role to a user without team id is throwing an error.

![image](https://user-images.githubusercontent.com/5024049/132939156-7ce63883-ebaf-4a59-8dbf-b2433e1452a3.png)

![image](https://user-images.githubusercontent.com/5024049/132939159-9800bf69-5ce1-4058-bac1-f0226d1ec51b.png)

What could be the issue?

**Comment by navaer** [comment]
I had exactly the same situation, for seeding a table this is what I did.
 
$team = Team::create(['user_id' => 1, 'name' => 'My Team', 'personal_team' => 1]);
        app(\Spatie\Permission\PermissionRegistrar::class)->setPermissionsTeamId($team->id);
        $user = User::create([
            'name' => 'User',
            'email_verified_at' => Carbon::now(),
            'email' => 'user@email.com',
            'password' => bcrypt('P@ssw0rd'),
            'current_team_id' => 1
        ]);
$user->assignRole(['role' => 'super-admin']);

**Reply by erikn69** [reply]
See [#1840](https://github.com/spatie/laravel-permission/discussions/1840#discussioncomment-1277282)

**Comment by erikn69** [comment]
Global role means you can use the same role on different teams
See [#1840](https://github.com/spatie/laravel-permission/discussions/1840#discussioncomment-1277282), maybe duplicate
